manifest.py

```diff
@@ -126,8 +129,25 @@ def convert(self, value, param, ctx):
     account = MbxAccountParamType()
 
 
+def validate_sources(sourcefile):
+    """Validate scheme and uniqueness of sources"""
+    sources = list([name.strip() for name in sourcefile])
+
+    # Validate scheme.
+    non_s3 = [name for name in sources if not name.startswith('s3://')]
+    if len(non_s3) > 0:
+        raise click.BadParameter("Sources {!r} do not have the required 's3' scheme.".format(non_s3))
+
+    # Identify duplicate sources.
+    dupes = [name for (name, count) in Counter(sources).items() if count > 1]
+    if len(dupes) > 0:
+        raise click.BadParameter("Duplicated sources {!r} cannot be processed.".format(dupes))
+
+    return sources
```

Comment:
Full tests are out of scope right now, but here's what I'm doing to check @vincentsarago 

### success

```
$ cat << EOF | python manifest.py --tileset "foo.x" --license "x" --account "foo" --product "x" --date "2018"
> s3://foo/bar
> s3://foo/bur
> {"sources": ["s3://foo/bar", "s3://foo/bur"], "info": {"tilesets": ["foo.x"], "license": "x", "account": "foo", "product": "x", "notes": "", "date": "2018"}}
```

### duplicate source failure

```
$ cat << EOF | python manifest.py --tileset "foo.x" --license "x" --account "foo" --product "x" --date "2018"
> s3://foo/bar
> s3://foo/bar
> Usage: manifest.py [OPTIONS] [SOURCEFILE]

Error: Invalid value: Duplicated sources ['s3://foo/bar'] cannot be processed.
```